From eddcd000bc89c1c9c17e1db31a20a655c51a064d Mon Sep 17 00:00:00 2001
From: Razvan Stefanescu <razvan.stefanescu@nxp.com>
Date: Fri, 18 Aug 2017 12:34:24 +0300
Subject: [PATCH 167/767] staging: dpaa2-evb: Update netlink parsing parameters

Accommodate extended ACL reporting addded to netlink parsing functions.

Signed-off-by: Razvan Stefanescu <razvan.stefanescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/evb/evb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/evb/evb.c b/drivers/staging/fsl-dpaa2/evb/evb.c
index d9df1796f32d..23d32dad73df 100644
--- a/drivers/staging/fsl-dpaa2/evb/evb.c
+++ b/drivers/staging/fsl-dpaa2/evb/evb.c
@@ -531,7 +531,7 @@ static int evb_setlink(struct net_device *netdev,
 	attr = nlmsg_find_attr(nlh, sizeof(struct ifinfomsg), IFLA_AF_SPEC);
 	if (attr) {
 		err = nla_parse_nested(tb, IFLA_BRIDGE_MAX, attr,
-				       ifla_br_policy);
+				       ifla_br_policy, NULL);
 		if (unlikely(err)) {
 			netdev_err(netdev,
 				   "nla_parse_nested for br_policy err %d\n",
@@ -737,7 +737,7 @@ static int evb_dellink(struct net_device *netdev,
 	if (!spec)
 		return 0;
 
-	err = nla_parse_nested(tb, IFLA_BRIDGE_MAX, spec, ifla_br_policy);
+	err = nla_parse_nested(tb, IFLA_BRIDGE_MAX, spec, ifla_br_policy, NULL);
 	if (unlikely(err))
 		return err;
 
-- 
2.17.0

